builtin/logical/aws/path_static_roles.go

```diff
@@ -219,6 +219,10 @@ func (b *backend) pathStaticRolesWrite(ctx context.Context, req *logical.Request
 		if err != nil {
 			return nil, fmt.Errorf("expected an item with name %q, but got an error: %w", config.Name, err)
 		}
+		// check if i is nil to prevent panic
```

Comment:
nit: I think this comment could be more useful if it explained why we might expect a nil value here. In particular, we seem to have an issue with repopulating the queue on reloads.

Reply:
Sorry to nitpick again but I think I prefer the previous error message. It doesn't actually matter though because PopByKey never returns an error.

Mostly my thinking was to make the comment more informative for developers that read the code in the future.

Reply:
No problem with the nitpicking, it's actually got me thinking we might want to visit adding an error at some point because no error returning it will never trigger that specific error, as far as I can tell. Not in this PR, but definitely in the short future.

Reply:
Sounds good! In that case, I propose we update the comment to the above suggestion.